test/e2e/index.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { chromium, firefox } from 'playwright'
+import { BASE_URL, CONTENT_SELECTOR, PAGE_TITLE, TITLE_SELECTOR } from './constants'
+
+test('render ./ from cms', async () => {
+  for (const browserType of [chromium, firefox]) {
+    const browser = await browserType.launch()
+    const page = await browser.newPage()
+
+    await page.goto(BASE_URL)
+
+    expect(await page.title()).toBe(PAGE_TITLE)
+    expect(await page.textContent(TITLE_SELECTOR)).toBe('index')
```

Comment:
Wondering if we should be using [selectors](https://playwright.dev/docs/selectors) here to get the content by value rather than by CSS class so that in the case that the generated class name changes the test doesn't break 😁

Reply:
Oh, damn it. Implementation details! Thank you, Reuben.